src/module/todo.js

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable quotes */
```

Comment:
- [ ] Kindly avoid disabling linters because linters make your code follow best practices and standards which are defined by developers.